launch.py

```diff
@@ -69,6 +69,18 @@ def bool_str(text):
     else:
         config["a2s"] = None
 
+    if env_defined("RCON_ADDRESS") and env_defined("RCON_PORT"):
+        if not env_defined("RCON_PASSWORD"):
+            config["rcon"]["password"] = ""
+        config["rcon"] = {
```

Comment:
Is there a concern with RCON being setup without a password?

Reply:
No concern, RCON does not start without a password :) Tested it myself to be sure - the server starts, but RCON does not 

https://community.bistudio.com/wiki/Arma_Reforger:Server_Config#password

Reply:
```py
if not env_defined("RCON_PASSWORD"):
    config["rcon"]["password"] = ""
````
This is not doing anything then, right?
Since below it is still reading from os.environ